mood-analysis.rb

```diff
@@ -19,6 +19,52 @@ def analyze_mood(words)
   return ":-|"
 end
 
+def strip_punctuation(sentence)
+  sentence.gsub(/[[:punct:]]/, "")
+end
```

Comment:
Oooh.. this is neat. I'm assuming :punct: is defined by ruby or regex or something? I just manually removed mine, but this is definitely a better solution.